sdk/storagemover/arm-storagemover/CHANGELOG.md

```diff
@@ -1,15 +1,38 @@
 # Release History
+    
+## 2.0.0-beta.1 (2023-07-12)
+    
+**Features**
 
-## 1.0.1 (Unreleased)
+  - Added Interface AzureKeyVaultSmbCredentials
+  - Added Interface AzureStorageSmbFileShareEndpointProperties
+  - Added Interface AzureStorageSmbFileShareEndpointUpdateProperties
+  - Added Interface Credentials
+  - Added Interface SmbMountEndpointProperties
+  - Added Interface SmbMountEndpointUpdateProperties
+  - Added Type Alias CredentialsUnion
+  - Added Type Alias CredentialType
+  - Added Type Alias EndpointBaseUpdatePropertiesUnion
+  - Interface Resource has a new optional parameter systemData
+  - Type of parameter endpointType of interface EndpointBaseProperties is changed from "AzureStorageBlobContainer" | "NfsMount" to "AzureStorageBlobContainer" | "NfsMount" | "AzureStorageSmbFileShare" | "SmbMount"
+  - Added Enum KnownCredentialType
+  - Enum KnownEndpointType has a new value AzureStorageSmbFileShare
+  - Enum KnownEndpointType has a new value SmbMount
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+**Breaking Changes**
 
+  - Interface Agent no longer has parameter systemData
+  - Interface Endpoint no longer has parameter systemData
+  - Interface JobDefinition no longer has parameter systemData
+  - Interface JobRun no longer has parameter systemData
+  - Interface Project no longer has parameter systemData
+  - Interface StorageMover no longer has parameter systemData
```

Comment:
Could you help check this above items are real breakings?

Reply:
After check, this parameter is changed from a subclass to a parent class, which should be non-breaking